### ISSUE TITLE
[AdaptiveCards- Schema Explorer]: Incorrect Role is announced by the screen reader for the 'Feedback link'.

### ISSUE THREAD

**Issue by Susmitha474** [microsoft/AdaptiveCards#7646]
[###](url) Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

Test Environment:

OS: Version 21H2 (OS Build 22000.318)

URL: [https://adaptivecards.io/](https://nam06.safelinks.protection.outlook.com/?url=https://adaptivecards.io/&data=04%7c01%7cv-ibalam%40microsoft.com%7cbd35d875ec3f4539762308d88f709775%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637417061036711036%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=dUolQbKGi1J8gsoxC2JG5TGoNi41msPfIIjKSIyPGHM%3D&reserved=0)

Edge Dev: version 105.0.1300.0 (Official build) dev (64-bit)

Screen reader: Narrator

Repro Steps:

1. Open above URL with valid credential.

2. Turn on the Narrator using ctrl+win+enter key.

3. Navigate to 'Schema Explorer' link and using arrow keys press enter key to activate it.

4. Navigate to 'Feedback' link using down arrow and observe the issue.

 

Actual Result:

Narrator is announcing 'Feedback Link' as 'Feedback Group'.

Expected Result:

Screen Reader should not convey the irrelevant information for the 'Feedback' control, it should announce it as 'Feedback Link'.

Note:

NVDA screen reader is not announcing the role of the 'Feedback' control as 'Link' its announcing as 'Feedback'.

User Experience:

It will impact the screen reader user if the screen reader does convey the irrelevant information for the links.

MAS Reference:

(https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/04.01.02)

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

![Feed back link](https://user-images.githubusercontent.com/104482612/177573157-2f76a33a-e4e6-4fed-a884-b53c84d8aed8.png)

https://user-images.githubusercontent.com/104482612/177288404-2698c648-fbb8-45c3-833d-a6c82d7fbede.mp4

### Card JSON

```JSON
NA
```

### Sample Code Language

NA

### Sample Code

NA

**Comment by vagpt** [comment]
Issue is not repro at our end, hence adding A11yTTValidated tag.
